qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
Oh I know, I'm just trying to make sure this is consistent across the code base and not a change that happens or not depending on who reviewed the code.